tools.go

```diff
@@ -0,0 +1,14 @@
+//+build tools
+
+package tools
+
+// Package tools is used to import go modules that we use for tooling as dependencies.
+// For more information, please refer to: https://github.com/go-modules-by-example/index/blob/master/010_tools/README.md
+
+import (
```

Comment:
Should we also include things like `golangci-lint` here?

Reply:
>Install from Source
Note: such go get installation aren't guaranteed to work. We recommend using binary installation. 

https://golangci-lint.run/usage/install/#install-from-source